HoneyBadgers.ConsoleApp/Repositories/UserRepository.cs

```diff
@@ -100,5 +100,65 @@ private string StringValidation(int minLength, int maxLength)
 
             return input;
         }
+        public void PrintUsers()
+        {
+            Console.WriteLine("LIST OF USERS:");
+            foreach (var user in Users)
+            {
+                Console.ForegroundColor = ConsoleColor.Green;
+                Console.WriteLine(
+                    $"ID: {user.Id} || FIRSTNAME: {user.FirstName} || LASTNAME: {user.LastName} || EMAIL: {user.Email}");
```

Comment:
```
Console.WriteLine($"| {"Name",-15} | {"Surname",-15} | {"MailAddress",-18} | {"Age",-5} | {"isActive",-8} | {"Books",-5} |");
Console.WriteLine(new String('=', 83))
Console.WriteLine($"| {item.Name,-15} | {item.Surname,-15} | {item.MailAddress,-18} | {item.Age,-5} | {item.isActive,-8} | {item.BookBorrowed.Count,-5} |");
```

Możesz to mniej więcej w ten sposób spróbować, w formie tabeli, nie powinno się rozjeżdzać i będzie ciut bardziej czytelne chyba (kopiowałem z pracy domowej, trzeba dane zmienić 😄) ale nie wiem czy nie szkoda czasu już na takie zabawy

Reply:
To do przemyślenia na webówce, brakuje już trochę czasu na to. Skupiamy się teraz na funkcjonalności.